Probesys/bileto
# imp: Disallow changing status if ticket status is finished

Related to https://github.com/Probesys/bileto/issues/127

Changes proposed in this pull request:

- Add style to disabled select
- Add more "color" styles to text
- Add a class to center content vertically
- Don't accept status and solution parameters if ticket status is finished (i.e. resolved or closed)
- Disable the solution checkbox and hide the status select (replaced by text) if ticket status is finished

How to test the feature manually:

1. Open a ticket, then resolve it (or close it)
2. Verify that the "solution" checkbox is disabled and the status select box is hidden

Pull request checklist:


_If you think one of the item isn’t applicable to the PR, please check it
anyway and precise `N/A` next to it._
